jetty-quickstart/src/main/java/org/eclipse/jetty/quickstart/QuickStartConfiguration.java

```diff
@@ -131,19 +145,23 @@ public void preConfigure(WebAppContext context) throws Exception
                 {
                     if (LOG.isDebugEnabled())
                         LOG.debug("No quickstart xml file, starting webapp {} normally", context);
-                    super.preConfigure(context);
```

Comment:
All derived `Configuration` classes call their `super()` methods, so leave it in.

Reply:
The assumption that all derived `Configuration` classes cal their super is incorrect, see other review comment below for actual list.